app/backend/routes/venues.js

```diff
@@ -10,4 +13,59 @@ router.get('/', (req,res,next) => {
   res.send(output)
 })
 
+router.post('/verify',ensureAuthenticated, (req, res, next) => {
```

Comment:
At this point, do you mind refactoring this method into own controller and documented. This route file is doing a bit too much and is going to cause future confusion. I believe you alluded a bit to this.